internal/cmd/http.go

```diff
@@ -146,6 +152,7 @@ func NewHTTPServer(
 		r.Mount("/api/v1", api)
 		r.Mount("/evaluate/v1", evaluateAPI)
 		r.Mount("/internal/v1", evaluateDataAPI)
+		r.Mount("/internal/analytics/v1", analyticsAPI)
```

Comment:
```suggestion
		r.Mount("/analytics/v1", analyticsAPI)
```

maybe?

or.. we could do:

```
/internal/v1/analytics
```

and just mount it above the `/internal/v1/` route so it matches first

Reply:
@markphelps That is cool with me! We can do that